JM/CF/Debugger/3_Game/CF_Debugger/CF_Debugger_Block.c

```diff
@@ -0,0 +1,144 @@
+class CF_Debugger_Entry
+{
+	string Key;
+	string Value;
+};
+
+class CF_Debugger_Block: ScriptViewTemplate<CF_Debugger_Block_Controller>
+{
+	private string m_Name;
+
+	private Object m_Target;
+	private bool m_HasTarget;
+
+	private bool m_Buffered;
+	private bool m_SwapClears;
+
+	private int m_Count;
+
+	private autoptr map<string, int> m_EntryMap;
```

Comment:
Why an array index map to access the values instead of just a string, value map?